### DISCUSSION TITLE
Memory utilization proces/subprocesss and out of process...

### DISCUSSION THREAD

**Discussion by DynConcepts** [cefsharp/CefSharp#4666] [General]
Yes, Cef uses external "CefSharp.BrowserSubprocess" for each instance... This shouldreduce the memory load on the "application process" (which is unfortunately huge and 32 bit, so memory is precious)..... 

Then I found the "OutOfProcess" project/repository... While it is currently broken, I was able to fix it sufficiently for my needs of evaluation and I have it working.. So I see the Example Process, the OutOfProcess.BrowserProcess) and the expected Cef processes...

This brings me to my question... How much does this reduce overall memory usage in the primary process, compared to using Cef "normally"....

**Comment by amaitland** [comment]
> While it is currently broken,

The project hasn't been updated for a while, Pull requests with updates are welcome.

> How much does this reduce overall memory usage in the primary process, compared to using Cef "normally"....

I've never had need to profile the difference. In theory it should be a fraction as you only need to load a smallish number of managed dlls. Memory usage would entirely depend on your usage scenario.

If you run the browser out of process you can run a `64bit` browser in a `32bit` process.
